filterapi/filterconfig.go

```diff
@@ -137,9 +137,18 @@ type VersionedAPISchema struct {
 type APISchemaName string
 
 const (
-	APISchemaOpenAI      APISchemaName = "OpenAI"
-	APISchemaAWSBedrock  APISchemaName = "AWSBedrock"
+	// APISchemaOpenAI represents the standard OpenAI API schema.
+	APISchemaOpenAI APISchemaName = "OpenAI"
+	// APISchemaAWSBedrock represents the AWS Bedrock API schema.
+	APISchemaAWSBedrock APISchemaName = "AWSBedrock"
+	// APISchemaAzureOpenAI represents the Azure OpenAI API schema.
 	APISchemaAzureOpenAI APISchemaName = "AzureOpenAI"
+	// APISchemaGCPGemini represents the Google Cloud Gemini API schema.
+	// Used for Gemini models hosted on Google Cloud Vertex AI.
+	APISchemaGCPGemini APISchemaName = "GCPGemini"
```

Comment:
this should be GCPVertexAI as defined here https://github.com/envoyproxy/ai-gateway/blob/main/api/v1alpha1/api.go#L458